src/firmware_new/boards/frankie_v1/io/drivetrain.h

```diff
@@ -1,29 +1,88 @@
 #pragma once
 
+#include "firmware/shared/math/matrix.h"
 #include "firmware_new/boards/frankie_v1/io/drivetrain_unit.h"
 
+typedef struct Drivetrain Drivetrain_t;
 /**
  * Initialize the IO layer for the Drivetrain
  *
  * NOTE: This does not take ownership of the given drive units. The driveunit's must
  *       remain valid until this function is called again.
  *
- * @param [in] front_left_drive_unit
- * @param [in] front_right_drive_unit
- * @param [in] back_left_drive_unit
- * @param [in] back_right_drive_unit
+ * @param front_left_drive_unit [in]
+
+ * @param front_right_drive_unit [in]
+
+ * @param back_left_drive_unit [in]
+
+ * @param back_right_drive_unit [in]
+
+ * @param robot_wheel_geometry [in] Wheel matrix representing the geometric
```

Comment:
What format is the matrix / data? Just reading the comment I have no idea how to set up this matrix if I needed to provide it.